src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -22,11 +23,48 @@
 from WMCore.WMSpec.WMWorkloadTools import loadSpecClassByType, setArgumentsWithDefault
 from WMCore.Cache.GenericDataCache import GenericDataCache, MemoryCacheStruct
 
+
 def workqueue_stat_validation(request_args):
     stat_keys = ['total_jobs', 'input_lumis', 'input_events', 'input_num_files']
```

Comment:
It looks like you forgot to update this code with the new constant variable.

Reply:
what do you mean?

Reply:
correct